templates/common/infra/bicep/core/ai/cognitiveservices.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'To create an openai account.'
```

Comment:
Format should be...

"Creates a..."

Reply:
And this is Cognitive Services, not just OpenAI

Reply:
Fixed.